openmc/material.py

```diff
@@ -445,31 +497,44 @@ def get_material_xml(self):
 
 
 class MaterialsFile(object):
+    """Materials file used for an OpenMC simulation. Corresponds directly to the
+    materials.xml input file.
 
-    def __init__(self):
+    Attributes
+    ----------
+    default_xs : str
+        The default cross section identifier applied to a nuclide when none is
+        specified
 
+    """
+
+    def __init__(self):
         # Initialize MaterialsFile class attributes
         self._materials = []
         self._default_xs = None
         self._materials_file = ET.Element("materials")
 
-
     @property
     def default_xs(self):
         return self._default_xs
 
-
     @default_xs.setter
     def default_xs(self, xs):
-
         if not is_string(xs):
             msg = 'Unable to set default xs to a non-string value'.format(xs)
             raise ValueError(msg)
 
         self._default_xs = xs
 
-
     def add_material(self, material):
+        """Add a material to the file.
+
+        Parameters
+        ----------
+        material : Material
```

Comment:
With no module mentioned, it is to be assumed the class is in the current module. autodoc understands this and will put in the hyperlink appropriately.